### PR TITLE
Improve weak ai base

### PR DESCRIPTION
**Pull request by Brett208**
I was unable to test on my local machine. Should be run with Outpost 2 before merging to check building placement and smelter route is working.

### CODE REVIEW

File: WeakAIBase.cpp [hunk 1]
```diff
@@ -106,10 +109,12 @@ void BuildAIBase(PlayerNum  aiPlayerNum, const LOCATION& initBaseLoc)
 	currentLoc.x += + 3;
 	CreateAIBuilding(unit, mapTokamak, currentLoc, aiPlayerNum, map_id::mapNone, weakAiBuildings);
 
-	createGuardPostCluster(aiPlayerNum, LOCATION(88 + X_, 127 + Y_), weakAiBuildings);
-	createGuardPostCluster(aiPlayerNum, LOCATION(80 + X_, 118 + Y_), weakAiBuildings);
-	createGuardPostCluster(aiPlayerNum, LOCATION(85 + X_, 138 + Y_), weakAiBuildings);
-	createGuardPostCluster(aiPlayerNum, LOCATION(80 + X_, 152 + Y_), weakAiBuildings);
+	CreateGuardPostCluster(aiPlayerNum, LOCATION(88 + X_, 127 + Y_), weakAiBuildings);
+	CreateGuardPostCluster(aiPlayerNum, LOCATION(81 + X_, 127 + Y_), weakAiBuildings);
```
Comment (WoodyEric) [inline WeakAIBase.cpp]: line 113 change X value to 68

Comment (WoodardEric) [inline WeakAIBase.cpp]: Line 113 change x value to 68 and y value to 118

File: WeakAIBase.cpp [hunk 2]
```diff
@@ -106,10 +109,12 @@ void BuildAIBase(PlayerNum  aiPlayerNum, const LOCATION& initBaseLoc)
...
+	CreateGuardPostCluster(aiPlayerNum, LOCATION(80 + X_, 118 + Y_), weakAiBuildings);
+	CreateGuardPostCluster(aiPlayerNum, LOCATION(85 + X_, 138 + Y_), weakAiBuildings);
+	CreateGuardPostCluster(aiPlayerNum, LOCATION(80 + X_, 152 + Y_), weakAiBuildings);
+	CreateGuardPostCluster(aiPlayerNum, LOCATION(77 + X_, 152 + Y_), weakAiBuildings);
```
Comment (WoodardEric) [inline WeakAIBase.cpp]: Line 117 change x value to 66

**Review by WoodardEric — COMMENTED** [review]
Guard post are overlapping buildings so I recommend some coordinate changes. Mining groups look good.

**Pull request merged**